src/main/webapp/script.js

```diff
@@ -78,10 +85,15 @@ function deletePotentialMatchInfo() {
   document.getElementById("carousel-indicators").innerHTML = '';
 }
 
-function createSlideshowElement(blobkey, className, name, bio) {
+function createSlideshowElement(blobkey, className, name, bio, noMatch) {
```

Comment:
nit: rather than passing in a boolean here, you could just pass in the image element, i.e. the caller would invoke either createImgElement or createImageFromBlobStore. Makes it a bit more clear what the parameter represents, and you'd also remove the blobKey parameter from this function (passing "images/no_image.png" as a blob key in the no-match case is a bit confusing, since that's not actually a blob key).